ci/pipelines/toolbox-staging-deploy.yml

```diff
@@ -43,22 +68,30 @@ resource_types:
     source:
       repository: concourse/registry-image-resource
       tag: "1.1.0"
-
+groups:
+  - name: pipeline-meta-update
+    jobs:
+      - update-toolbox-staging-ecr-pipeline
+  - name: toolbox
+    jobs:
+      - deploy-toolbox-to-staging
+      - smoke-test-toolbox-on-staging
+      - push-toolbox-to-production-ecr
 jobs:
   - name: update-toolbox-staging-ecr-pipeline
     plan:
       - get: toolbox-staging-deploy-config
         trigger: true
       - set_pipeline: toolbox-staging-deploy-config
         file: toolbox-staging-deploy-config/ci/pipelines/toolbox-staging-deploy.yml
-  - name: toolbox-staging-deploy
+  - name: deploy-toolbox-to-staging
     plan:
-      - get: toolbox-staging-deploy-config
       - get: toolbox-ecr-registry-staging
         trigger: true
-        params:
-          format: oci
+      - get: pay-infra
```

Comment:
No need to change just a passing comment, I found myself looking where `pay-infra` is used which is within `deploy-app.yml` so its not obvious until you follow some threads or just reason that its running terraform and that's in `pay-infra` :)

Reply:
hmm yeah, any ways we could make this clearer? Like specify inputs in main pipeline definition or something?